frontend/src/react-app-env.d.ts

```diff
@@ -1 +0,0 @@
-/// <reference types="react-scripts" />
```

Comment:
A co w koncu robi ten plik?

Reply:
Nie wiem :D

Reply:
No to teraz nie wiemy czy potrzebny czy nie :) Jak dla mnie skoro create-react-app go nam dał, to jakaś funkcje pełni. Dobrze byłoby wiedzieć jaka i czy ona nam jest niepotrzebna. Trzeba się nauczyć researchowac, Google nie gryzie a 1-3 pierwsze wyniki powinni dać nam satysfakcjonująca odpowiedź :)

Reply:
Trochę tego nie rozumiem. Ale prawdopodobnie przez to miałam problemy z tsconfig i testami

https://github.com/facebook/create-react-app/issues/6560

Reply:
No z tego co widzę, to brak tego może rodzić problemy. Ale warto o tym pamiętać, jakby coś wystąpiło. Z tego co widzę na developie przechodzą testy na GitHub Actions. Więc może miałaś problem ze swoim środowiskiem?